shared/build.gradle.kts

```diff
@@ -29,13 +35,37 @@ kotlin {
     }
     
     sourceSets {
-        val commonMain by getting
+        val commonMain by getting {
+            dependencies {
+
```

Comment:
redundant line? or adding a `// Core` comment?

Reply:
Fixed in https://github.com/suho/kmm-ic/pull/86/commits/267561ece685f282ba111bf3132e937820df7d04